### PR TITLE
My plan: update free plan page

### LINKED ISSUE #10995: Jetpack Plans and My Plans areas: Consolidate up-sell prompts
If a customer is on Jetpack Free, the "My Plan" and "Plans" area offers up a bunch of up-sell CTAs in a tacky sort of way. I think we should come up with a single up-sell CTA area that shuffles content based on the context of the customer, otherwise this looks like complete spam and bloated.

**Currently:**
fireshot capture 003 - jetpack cathol_ - https___catholic-tangle jurassic ninja_wp-admin_admin php
fireshot capture 004 - jetpack cathol_ - https___catholic-tangle jurassic ninja_wp-admin_admin php

**Proposed:**

Just some quick in browser mocks that need a little more thought and discussion...

**"My Plan"**

fireshot capture 006 - jetpack cathol_ - https___catholic-tangle jurassic ninja_wp-admin_admin php

* Add value cards for customers on free, with more strategically placed upgrade prompts.
* Right now the cards displayed on "free" don't accurately reflect all the features included with free - it's not clear those are upgrades since there's no CTA to help them along. I think we need to sell more people on the value of the feature rather than just arbitrarily asking them to upgrade from a small white square box, so I propose adding buttons and taking them to the proper areas within the plugin

Content not finalized!

> **Design the perfect website**
> 
> _text about customization and themes tbd...._
>
> Button: [Explore design tools] url: https://jetpack.com/features/design/
>
> **Increase traffic and revenue**
>
> _text about our marketing and growth tools tbd...._
> 
> Button: [Configure marketing & traffic tools] url: settings: traffic
>
> **Always-on security to keep you safe**
>
> _text about our security tools tbd...._
>
> Button: [Ensure you're safe] url: settings: security
>
> **Built-in performance**
>
> _text about performance tbd...._
>
> Button: [Speed up your site] url: settings/performance (coming soon)
>
> **Mobile apps for any device**
>
> _text about our mobile apps tbd...._
>
> Button: [Get the WordPress app] url: https://apps.wordpress.com/
>
> **Enjoy priority support**
> 
> _text about priority support tbd...._
>
> Button: [Get faster solutions] url: Plans tab
>
> **Jetpack offers so much more**
> 
> _text about more benefits tbd...._
>
> Button: [Explore all features] url: Plans tab

I want to keep customers in the jetpack UI where possible, but since our product and marketing IA in inconsistent, it's difficult at the moment. Some links will divert customers to settings areas, while other areas will take them to jetpack.com (example: design & themes)

* I'm not sure how activity fits into this in a way that explains value to the customer on free, so I didn't include it 

**"Plans"**

screen shot 2018-12-17 at 4 40 32 pm

* The plans grid is the only up-sell area we need on this page - we can roll any other feature mention into this grid - such as the Premium themes, priority support, and WP Apps. We also need to clean up this plans table quite a bit to be consistent with Jetpack.com

### PR DESCRIPTION
**Pull request by keoshi**
Fixes #10995 

#### Changes proposed in this Pull Request:

* Update the Free plan page on My Plan:
  * Make plan feature cards more useful and actionable.
  * Reorder cards so that Security & Performance are the top two focuses.
  * Remove Premium plan promo.

#### Testing instructions:

* Go to your Jetpack site or [create a new one](https://jurassic.ninja/create/).
* Connect Jetpack if it isn't already.
* Go back to wp-admin and open the `My Plan` page.
* Make sure the page is cleaner from distractions, and that all cards have a corresponding action.

#### Before

image

#### After

image

#### Proposed changelog entry for your changes:

* My Plan page is now more useful and actionable for free plans.

### CODE REVIEW

**Comment by joanrho** [comment]
@keoshi thanks for starting the build on this! So far, it works great on testing. I'm not sure about the last card that condenses the 2-col wide paid plan upsell card @jeffgolenski originally had designed in his mockups in the Issue...so, let's discuss more on Monday.

Unrelated to this PR, but worth mentioning here while I'm testing: the many notifications that pop up on my first visit to this page (some new Coupon code thing?), Jumpstart, and everything below the Free plan features are distracting. I've made a note to address this as part of Improved Onboarding.

**Comment by jeherve** [comment]
> the many notifications that pop up on my first visit to this page (some new Coupon code thing?), Jumpstart, and everything below the Free plan features are distracting. I've made a note to address this as part of Improved Onboarding.

Related: #10752

**Comment by jeffgolenski** [comment]
This is a great start, thanks @keoshi We've got some time until the next release, so let's take our time to craft a little more on this page. Including conditionally removing all the added up-sells on this page, as mentioned in the originally issue. https://github.com/Automattic/jetpack/issues/10995  (just posting this here for others to see)

**Comment by keoshi** [comment]
@jeffgolenski I believe I removed all up-sells — which ones are you referring to?

**Comment by jeffgolenski** [comment]
@keoshi I'm still seeing the prompt to download the app in the footer. I suppose that's not technically an up-sell, but a cross-sell. Can we work that into one of the cards you think?... actually, do we have metrics on that CTAs success? 

screen shot 2019-03-04 at 12 33 23 pm


Also, we should take a look at fixing up the plans grid page as well. First off, just removing the giant themes up-sell banner (after we look at CR metrics, of course)

screen shot 2019-03-04 at 12 31 31 pm

**Comment by keoshi** [comment]
@jeffgolenski, thanks for the feedback!

> Can we work that into one of the cards you think? (...) do we have metrics on that CTAs success?

Yep, I think that's a good idea, but take a look at the data I sent you on Slack. The amount of clicks on that banner is non negligible, and the card seems to be pretty effective (as annoying as it might be).

> Also, we should take a look at fixing up the plans grid page as well.

Yep, let's do it on a separate PR though since this is for the My Plan page. Can you open a new issue for that, please?

**Comment by jeffgolenski** [comment]
@keoshi Per our chat, we can bump the Plans Page modifications to our Project 10. Jetpack Plans Marketing Improvements

**Comment by keoshi** [comment]
I've updated this PR @jeffgolenski @joanrho — would appreciate another quick design review.

File: _inc/client/my-plan/my-plan-body.jsx [hunk 1]
```diff
@@ -40,7 +41,7 @@ class MyPlanBody extends React.Component {
 
 	trackPlansClick = target => {
 		analytics.tracks.recordJetpackClick( {
-			page: 'plans',
+			page: 'my-plan',
```
Comment (jeherve) [inline _inc/client/my-plan/my-plan-body.jsx]: Nice catch!

File: _inc/client/my-plan/my-plan-header.jsx [hunk 2]
```diff
@@ -24,295 +23,25 @@ class MyPlanHeader extends React.Component {
...
-					);
-				}
 				planCard = (
 					<div className="jp-landing__plan-card">
 						<div className="jp-landing__plan-card-img">
 							<img
 								src={ imagePath + '/plans/plan-free.svg' }
 								className="jp-landing__plan-icon"
-								alt=""
+								alt={ __( 'Jetpack Free Plan' ) }
```
Comment (jeherve) [inline _inc/client/my-plan/my-plan-header.jsx]: Nice to add those in 👍

**Review by jeherve — APPROVED** [review]
I really like that change, and I don't see any issues with it. The page is so much more useful to me that way, I can finally go to "My Plan" and actually know what plan I am on! 🎉 	

🚢

File: _inc/client/my-plan/my-plan-header.jsx (hunk 2 above)
Comment (keoshi) [reply _inc/client/my-plan/my-plan-header.jsx]: 🎉

File: _inc/client/my-plan/my-plan-body.jsx (hunk 1 above)
Comment (keoshi) [reply _inc/client/my-plan/my-plan-body.jsx]: 🙌

**Review by joanrho — APPROVED** [review]
Just tested and looks/works as expected. Really nice work @keoshi, thank you! 🚢

**Pull request merged**